kitchensink/src/app/common/PageElements/FoundationElements/ColorRow/ColorRow.jsx

```diff
@@ -0,0 +1,167 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import ColorBox from '../ColorBox/ColorBox';
+import './colorRow.css';
+
+const ColorRow = (props) => (
+  <div styleName="colorRowContainer">
+    {
+      <table>
+        {
+          (props.showInfo && (props.showInfo === 'fullInfo' || props.showInfo === 'onlyTitles'))
+          && (
+            <thead>
+              <tr>
+                {
+                  props.title.length > 0 && <th key="base-col-header" style={{ width: (props.rowHeader.width ? props.rowHeader.width : 'auto'), textAlign: 'left' }} styleName={classNames('title', { [`${props.rowSize}`]: true })}>{props.title}</th>
+                }
+                {
+                  props.preColumns > 0
+                  && <th colSpan={props.preColumns}>&nbsp;</th>
+                }
+                {
+                  (props.rowHeader && Object.keys(props.rowHeader).length > 0)
+                  && props.rowData.map((element) => {
+                    if (props.showInfo && (props.showInfo === 'fullInfo' || props.showInfo === 'onlyTitles')) {
+                      const title = element.title.length > 0 ? element.title : '';
+                      return (
+                        <th key={`${element.colors[0].name}-title`}>
+                          <div styleName={classNames('title', { [`${props.rowSize}`]: true })}>{`${title}`}</div>
```

Comment:
here you do not need a template literal, since title is already a string : 
`>{title}</div>`

Reply:
✅